mlflow/protos/service.proto

```diff
@@ -1129,4 +1129,4 @@ message GetExperimentByName {
     // Experiment details.
     optional Experiment experiment = 1;
   }
-}
+}
```

Comment:
Can we revert changes to this file?

Reply:
i want and trying figure out how, since locally i dont see any change. I understand it is some line ending diff, but trying to figure out way to fix it.

Reply:
Done.